### PR TITLE
feat: Added shortcut button  [123]

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description

Added shortcut button in the collection page 

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@library/icons/FolderPlus.svelte [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_3439_4638)">
+<path d="M13.3327 11.3333C13.3327 12.0687 12.7348 12.6667 11.9993 12.6667H3.99935C3.26393 12.6667 2.66602 12.0687 2.66602 11.3333V4.66666C2.66602 3.93124 3.26393 3.33333 3.99935 3.33333H6.66602C7.08477 3.33333 7.4806 3.53124 7.73268 3.86666L8.13268 4.39999C8.25768 4.56874 8.4556 4.66666 8.66602 4.66666H11.9993C12.7348 4.66666 13.3327 5.26458 13.3327 5.99999V11.3333ZM7.49935 10.5C7.49935 10.7771 7.72227 11 7.99935 11C8.27643 11 8.49935 10.7771 8.49935 10.5V9.16666H9.83268C10.1098 9.16666 10.3327 8.94375 10.3327 8.66666C10.3327 8.38958 10.1098 8.16666 9.83268 8.16666H8.49935V6.83333C8.49935 6.55625 8.27643 6.33333 7.99935 6.33333C7.72227 6.33333 7.49935 6.55625 7.49935 6.83333V8.16666H6.16602C5.88893 8.16666 5.66602 8.38958 5.66602 8.66666C5.66602 8.94375 5.88893 9.16666 6.16602 9.16666H7.49935V10.5Z" fill="#8A9299"/>
```
Comment (itsmdasifraza) [inline src/packages/@library/icons/FolderPlus.svelte:3]: <script>
    export let height = "24px";
    export let width = "24px";
    export let color = "grey";
  </script>
  
add this script, and pass value as a props

File: src/packages/@workspaces/features/collection-list/components/collection/Collection.svelte [hunk 2]
```diff
@@ -516,6 +521,40 @@
...
+            on:click={() => {
+              onItemCreated("requestCollection", {
+                workspaceId: collection.workspaceId,
+                collection,
+              });
+            }}
+          >
+            <RequestIcon height="16px" width="16px" />
+          </div>
+        </div>
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/collection-list/components/collection/Collection.svelte:557]: pass color prop

File: src/packages/@workspaces/features/collection-list/components/collection/Collection.svelte [hunk 3]
```diff
@@ -516,6 +521,40 @@
...
+            role="button"
+            on:click={() => {
+              onItemCreated("folder", {
+                workspaceId: collection.workspaceId,
+                collection,
+              });
+            }}
+          >
+            <FolderPlusIcon />
+          </div>
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/collection-list/components/collection/Collection.svelte:542]: pass height, width ,color prop here

File: src/packages/@workspaces/features/collection-list/components/collection/Collection.svelte [hunk 4]
```diff
@@ -680,4 +719,14 @@
...
+
+  .collection-sub-btn {
+    margin-left: 24px;
+  }
+  .folderIcon:hover{
+    background: var(--right-border);
+  }
+  .requestIcon:hover{
+    background: var(--right-border);
+  }
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/collection-list/components/collection/Collection.svelte:731]: use common class for both

**Review by itsmdasifraza — CHANGES_REQUESTED** [review]

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**